test/powershell/engine/PropertyAccessor.Tests.ps1

```diff
@@ -0,0 +1,96 @@
+#
+# Functional tests to verify basic conditions for IO to the PowerShellProperties.json files
+# The properties files are supported on non-Windows OSes, but the tests are specific to 
+# Windows so that file IO can be verified using supported cmdlets.
+# 
+
+try {
+
+    #skip all tests on non-windows platform
+    $originalDefaultParameterValues = $PSDefaultParameterValues.Clone()
+    $IsNotSkipped = ($IsWindows -eq $true);
+    $PSDefaultParameterValues["it:skip"] = !$IsNotSkipped
+    $ShouldSkipTest = !$IsNotSkipped
+
+    Describe "User-Specific PowerShellProperties.json Modifications" -Tags "CI" {
+
+        BeforeAll {
+            if ($IsNotSkipped) {
+                # Discover the user-specific PowerShellProperties.json file
+                $splitModulePath = $env:PSMODULEPATH -split ";"
+                $userModuleDir = $splitModulePath[0].TrimEnd("\")
+                $userSettingsDir = $userModuleDir -replace "\\Modules", ""
```

Comment:
Assuming the first module path is the user module path is fragile. This may not work when a dev is running the test with a profile. I think it's safer to construct the path to the user property.json file. That should be easy given that the test only runs on win-platform.